meta-resin-common/recipes-connectivity/networkmanager/files/NetworkManager.conf

```diff
@@ -0,0 +1,6 @@
+[main]
+dns=default
+plugins=keyfile
+
+[keyfile]
+unmanaged-devices=interface-name:docker0
```

Comment:
I don't really think that "it just works" is a good argument here :) What say is that tun devices are handled by another application so I don't want nm to interfere with that. Do we know that this doesn't happen?